.github/workflows/test.yml

```diff
@@ -21,6 +21,8 @@ jobs:
         uses: ./formatting
         with:
           path: coreMQTT
+          exclude-files: lexicon.txt
+          exclude-dirs: build;docs
```

Comment:
Should this be comma?

Reply:
Yes, good catch, have replaced with comma.